src/common/base/status.h

```diff
@@ -147,6 +147,15 @@ inline Status StatusAdapter<px::statuspb::Status>(const px::statuspb::Status& s)
 
 }  // namespace px
 
+// Other possible variations:
+// PX_RETURN_VALUE_IF_OK
+// PX_RETURN_STATUS_OR_IF_OK
```

Comment:
Are these suggestions for other possible names for this macro? They seem to articulate slightly different behavior and seems like something that can be removed.

Reply:
Those are suggestions for different macros with different behavior, but agreed about removing. Doing that now.